meta-resin-common/recipes-connectivity/resin-proxy-config/resin-proxy-config/redsocks.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=redsocks transparent proxy redirector
+Requires=mnt-boot.mount resin-proxy-config.service
+After=mnt-boot.mount resin-proxy-config.service
+
+[Service]
+ExecStart=@BASE_BINDIR@/sh -c "source /usr/sbin/resin-vars && \
```

Comment:
I don' think this complication is needed. We can just let resin-proxy-config.service fail which will make this service not start as well when there is no configuration to transport from boot partition. So I would let "if [ ! -f $REDSOCKSCONF ]; then" exit with error which will block starting this service.